LDH/Day3/문자열 다루기 기본.js

```diff
@@ -0,0 +1,12 @@
+function solution(s) {
+    if (s.length === 4 || s.length === 6) {
+        for (let i = 0; i < s.length; i++) {
+            if (isNaN(s[i])) {
+                return false;
+            }
+        }
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
```suggestion
if (s.length !== 4 && s.length !== 6) return false;

for (let i = 0; i < s.length; i++) {
  if (isNaN(s[i])) return false;
}

return true;
```
이런식으로 조건문을 최적화 할 수도 있을거 같네요!